inception-project/inception
# #1669 - Curation won't start properly

**What's in the PR**
- Switch to newer WebAnno version where the slow and buggy "invisible annotations" field has been removed from the curation page

**How to test manually**
* Make an annotation which crosses two sentences
* Finish the document so you can open it in the curation page
* Configure the page to show only one sentence at a time
* Observe that there is no panel "invisible annotations on other pages" where the invisible cross-sentence annotation is mentioned

**Automatic testing**

**Documentation**
